proto/sifnode/ethbridge/v1/types.proto

```diff
@@ -35,35 +35,33 @@ message EthBridgeClaim {
   ClaimType claim_type = 10;
 }
 
-message PeggyTokens {
-  repeated string tokens = 1;
-}
+message PeggyTokens { repeated string tokens = 1; }
 
 // Claim type enum
 enum ClaimType {
   // Unspecified claim type
   CLAIM_TYPE_UNSPECIFIED = 0;
-  // Lock claim type
-  CLAIM_TYPE_LOCK = 1;
   // Burn claim type
-  CLAIM_TYPE_BURN = 2;
+  CLAIM_TYPE_BURN = 1;
+  // Lock claim type
+  CLAIM_TYPE_LOCK = 2;
```

Comment:
I think we need to look into what's happening here a bit more closely, probably especially around export / import and string serialisation. Would be good to see what our unit tests are or should be asserting here.

Was this required as a fix for getting transfers to work?

Reply:
is there a reason why the order is being changed? Changing the order of protobuf data elements shouldn't happen once they've been created, though I'm sure you know this.

Reply:
@marshsif this is values for an enum, not ordering of fields in a message

Reply:
@timlind Yes, this was required for getting transfers to work.  I believe this proto file has to match our Solidity code since this the ClaimType used in EthBridgeClaim.  I might be confused about this though.

Reply:
And this did need the test changes that were in feature/peggy42, I pulled those in